client.go

```diff
@@ -28,9 +28,10 @@ import (
 )
 
 type EntityType string
-const(
-	PORT_GROUP EntityType = "PORT_GROUP"
-	LOGICAL_SWITCH EntityType = "LOICAL_SWITCH"
+
```

Comment:
fmt should be part of different commit.

Reply:
@noah8713 these fmt changes aren't needed with this commit?

Reply:
no